packages/contracts/contracts/voting/multisig/Multisig.sol

```diff
@@ -49,6 +49,12 @@ contract Multisig is TimeHelpers, PluginUUPSUpgradeable, Addresslist {
         uint256 addresslistLength;
     }
 
+    /// @notice A container for the plugin settings
+    /// @param onlyListed Whether only members in the addresslist can create a proposal
+    struct PluginSettings {
+        bool onlyListed;
```

Comment:
It seems like `minApprovals_` is also `settings` kind of guy.  Maybe we can put them in the same struct ? Let's reconfirm with @Michael-A-Heuer

Reply:
good point. @Michael-A-Heuer  wdyt?

Reply:
Yes, it should also be in the struct. In `MajorityVotingBase`, we named it 
`struct VotingSettings`. 
Should it be named `struct ApprovalSettings` here?